### ISSUE TITLE
Assembly dependency version issues 2.1.0.0 vs 2.1.1.0

### ISSUE THREAD

**Issue by penfold** [unitycontainer/abstractions#15]
Hi,

Please can you release a consistent set of NuGet packages where they are all using Unity.Abstractions 2.1.1.0?

Unity.Container depends on Unity.Abstractions 2.1.1.0 but NuGet is only on Unity.Abstractions 2.1.0.0

Having a little version hell and fiddling the app.config/web.config dependentAssembly settings is breaking my head.

Thanks,
Paul

**Comment by ENikS** [comment]
I am working on releasing Abstractions.2.1.2 and the rest of packages depending on it. 
Would you like to help to speed it up? I would be happy to accept a pull request...

**Comment by ENikS** [comment]
If you are using individual packages such as MVC or ASP I would recommend not to mix them with ```Unity``` composite package. Use all individual packages Unity.Abstactions, Unity.Container, and etc. This way you would guarantee they all depend on the same thing.

**Comment by penfold** [comment]
Sorry, wasn't intended as a rant.

**Comment by ENikS** [comment]
I was kind of joking. Did you resolve the issue?

**Comment by penfold** [comment]
I had installed Unity.Containter in one project and Unity.Abstractions in another. All using the latest Nuget packages. The issue was that Unity.Container depended on a newer versions of Unity.Abstractions which wasn't available on Nuget.

I dropped back to an earlier 0.0.1 point release and the dependency versions all match up. It just looked like there was a missing Nuget package for Unity.Abstractions 2.1.1.0 to go along side the latest Unity.Container package.
